en/airframes_experimental/traxxas_stampede.md

```diff
@@ -0,0 +1,122 @@
+
+![Assembly1](../../assets/airframes/experimental/stampede/panel_top.png)
+
+![Assembly2](../../assets/airframes/experimental/stampede/panel_bottom.png)
+
+![Assembly3](../../assets/airframes/experimental/stampede/final_assembly.jpg)
+
+![Assembly4](../../assets/airframes/experimental/stampede/final_side.jpg)
+
+![Assembly5](../../assets/airframes/experimental/stampede/mounting_detail.jpg)
+
+For this particular mounting it was chosen to use the clip supplied with the rover to attach the upper plane. For this, two supports were 3D printed. The CAD files are provided [here](../../assets/airframes/experimental/stampede/plane_holders.zip).
+
+> **Warning** It is **HIGHLY RECOMMENDED** to set the ESC in training mode (see Traxxas Stampede Manual) so to reduce the power to 50%.
+
+
+## Output connections
+
+| PWM Output | Rate | Actuator |
+| -- | -- | -- |
+| MAIN1 | 50 Hz | Not connected |
+| MAIN2 | 50 Hz | Steering servo |
+| MAIN3 | 50 Hz | Not Connected |
+| MAIN4 | 50 Hz | ESC input |
+| MAIN5 | 50 Hz | Not Connected |
+| MAIN6 | 50 Hz | Not Connected |
+| MAIN7 | 50 Hz | Not Connected |
+| MAIN8 | 50 Hz | Not Connected |
+
+## Simulation
+
+To test the algorithms, a simulation was used. This can be ran by running 
+
+ ```sh
+ make posix gazebo-rover
+ ``` 
+
+ from the Firmware folder. 
+
+## Usage
+At the current state, the only supported modes are `MISSION`  and `MANUAL` when a RC remote is connected. To use the mission mode, first upload a new mission to the vehicle with QGC. Then, BEFORE ARMING, select `MISSION` and then arm. 
+
+> **Warning** It is **_VERY IMPORTANT_** to do a mission composed **_ONLY_** by normal waypoints (i.e. **_NO TAKEOFF WAYPOINTS_**) and it is crucial to **_SET THE WAYPOINT HEIGHT OF EVERY WAYPOINT TO 0_** for a correct execution. Failing to do so will cause the rover to continuously spin around a waypoint.
+
+
+A correct mission setup looks as follows
+
+![mission](../../assets/airframes/experimental/stampede/correct_mission.png)
+ 
+Since at the current state of development there is no obstacle avoidance, it is highly recommended to test missions in the simulator first to get acquainted with the software usage.
+
+## Software location
+
+As it still is a work in progress, the code can be obtained from a personal repository currently being considered for merging in PRs [7139](https://github.com/PX4/Firmware/pull/7139) and [7175](https://github.com/PX4/Firmware/pull/7175). 
+
+```sh
+cd ~ 
+mkdir src
+cd src
+git clone -b pr-rover-rebase git@github.com:marcozorzi/Firmware.git
+cd Firmware
+git submodule update --init --recursive
+```
+
+## Parameters
+
+The current software provides an attitude controller and a position controller. These build on top of the fixed wing framework to move an Ackerman steered vehicle following GPS coordinates (real or fake). In order to this, 2 PIDs were implemented, one for the steering and the other for the speed. 
```

Comment:
Perhaps be a bit more declarative - tell users that they need to set these parameters manually, and link to instructions in QGC for setting parameters